app/views/projects/index.phtml

```diff
@@ -15,34 +15,40 @@ $isAdmin = $this->currentUser && $this->currentUser->getRole() === 'admin';
                         </h5>
                     </a>
 
-                    <?php if ($isAdmin): ?>
-                    <div class="mt-3" style="position: relative; z-index: 1;">
-                        <a href="<?= route('projects.edit', ['id' => $project->getId()]) ?>" class="card-link text-decoration-none small">
-                            <i class="bi bi-pencil-square"></i> Editar
-                        </a>
+                    <?php if ($isAdmin) : ?>
+                        <div class="mt-3" style="position: relative; z-index: 1;">
+                            <a href="<?= route('projects.edit', ['id' => $project->getId()]) ?>" class="card-link text-decoration-none small">
+                                <i class="bi bi-pencil-square"></i> Editar
+                            </a>
 
-                        <form action="<?= route('projects.destroy', ['id' => $project->getId()]) ?>" method="POST" class="d-inline">
-                            <input type="hidden" name="_method" value="DELETE">
-                            <button type="submit" class="card-link text-decoration-none small text-danger btn btn-link p-0">
-                                <i class="bi bi-trash"></i> Excluir
-                            </button>
-                        </form>
-                    </div>
+                            <form action="<?= route('projects.destroy', ['id' => $project->getId()]) ?>" method="POST" class="d-inline">
+                                <input type="hidden" name="_method" value="DELETE">
+                                <button type="submit" class="card-link text-decoration-none small text-danger btn btn-link p-0">
+                                    <i class="bi bi-trash"></i> Excluir
+                                </button>
+                            </form>
+                        </div>
                     <?php endif; ?>
 
                 </div>
             </div>
         </div>
     <?php endforeach; ?>
 
-    <?php if ($isAdmin): ?>
-    <div class="col-12 col-md-6 col-lg-4 mb-4">
-        <a href="<?= route('projects.new') ?>" class="card h-100 card-add-new text-decoration-none">
-            <div class="card-body text-center d-flex flex-column justify-content-center align-items-center">
-                <i class="bi bi-plus-circle-dotted fs-1 text-secondary mb-2"></i>
-                <h5 class="card-title text-secondary">Cadastrar Projeto</h5>
-            </div>
-        </a>
-    </div>
+    <?php if ($isAdmin) : ?>
```

Comment:
Here should be
`$this->current_user()->is('admin')`
or
`$this->current_user()->isAdmin()`